models/index.js

```diff
@@ -26,6 +27,36 @@ Data.belongsTo(User, {
     onDelete: 'CASCADE'
 });
 
+// Forum Comments Relationships
+
+User.hasMany(Comment, {
+    through: {
+      model: Post,
+      unique: false
+    }
+})
+
+Comment.belongsTo(User, {
+    through: {
+      model: Post,
+      unique: false
+    }
+})
+
+Post.hasMany(Comment, {
+    through: {
+      model: User,
+      unique: false
+    }
+})
+
+Comment.belongsTo(Post, {
+    through: {
+      model: User,
+      unique: false
+    }
+})
```

Comment:
The "through" property only works with belongsToMany releation ship in M:M or M:N.
[https://sequelize.org/master/manual/advanced-many-to-many.html](url)